perf/flame.jl

```diff
@@ -28,6 +28,15 @@ prof = Profile.@profile begin
     end
 end
 
-import PProf
-PProf.pprof()
-# http://localhost:57599/ui/flamegraph?tf
+if !isempty(get(ENV, "CI_PERF_CPUPROFILE", ""))
+    import ChormeProfileFormat
+    output_path =
+        mkpath(get(ENV, "CI_OUTPUT_DIR", joinpath(@__DIR__, "output")))
+    testname = splitext(splitdir(filename)[end])[1]
+    cpufile = joinpath(output_path, testname * ".cpuprofile")
```

Comment:
I think the profile is usually saved as `profile.pb.gz`

Reply:
this is for the chrometrace format  extension: https://docs.google.com/document/d/1CvAClvFfyA5R-PhYUmn5OOQtYMH4h6I0nSsKchNAySU/edit

the writer just grabs the latest profile data from the system